packages/codegen/src/schema.ts

```diff
@@ -63,6 +63,7 @@ export const revlinkSchema = z.object({
 export const tableSchema = z.object({
   name: z.string(),
   columns: z.array(columnSchema),
+  primaryKey: z.array(z.string()).optional(),
```

Comment:
This information is returned in the `xata.ts` file after init is run for pgroll branches. I just tested it. For non pgroll branches I don't think we don't need hardcoding to `id` since this goes to next channel?